source/clear-linux/guides/tooling/mixer.rst

```diff
@@ -17,19 +17,20 @@ Description
 
 mixer uses the following sources as inputs to generate update content:
 
-* Upstream |CL| bundles with their corresponding RPM packages
+* Upstream |CL| bundles with their corresponding :abbr:`RPM (RPM Package Manager)`
+packages
```

Comment:
"packages" needs to left-align with "U" in upstream, above it, to resolve build error. Please fix.

Reply:
@mvincerx Fixed the indentation error. Checked html with make and received no errors.